ci-curriculum.md

```diff
@@ -0,0 +1,82 @@
+# CI Curriculum - Community Outreach Program
+
+These are the essential skills that every participant attending the Community Outreach Program must be familiar with. 
+We have built up this resource library by creating our own internal content and by curating our favorite blog posts, documentation sites and courses 
+from around the Harness Ecosystem.
+
+Here, you have access to resources which will help you understand the general concepts as well as the specifics of the Harness Architechture 
+in depth.
+
+### Harness Product Overview
+
+Harness Product(s): Use-cases, going through Harness Documentation, Tech Boot Camp Sessions.
+
+- Most common Use cases for Harness
+- Failure Strategies 
+- Overview of Application Objects
+- Walking through UI for each application object (service, workflow etc)
+
+### Harness NextGen 
+
+- Delegate
+- Git experience 
+
+
+### Harness CI
+
+- Reviewing a Pipeline and all the steps associated with it 
+- UI walkthrough (CI)
+- InputSets
+- Test Intelligence (Overview)
+- Test Intelligence with Parallelisim
+
+
+### CI - Learning Checklist (Udemy or Google)
+
+- [ ] Kubernetes
+- [ ] Docker
+- [ ] Running queries on Google Stackdriver
+
+### CI - Backend Checklist
+
+- [ ] Golang
+- [ ] Google Guice
+- [ ] Lombok
+- [ ] SpringBoot
+- [ ] Mongo queries
+- [ ] Java 8 streaming
+- [ ] Java
+- [ ] Redis
+
+
+### CI - Learning Checklist (Harness Specific)
+
+- [ ] How to setup PR env (Personal environment to test everything end to end) 
+
+- [ ] How deployment happens in Harness.
+
+- [ ] How to update configuration for deployment in QA.
+
+- [ ] How to create feature build Artifact.
+
+
+## Resources 
```

Comment:
Let's just mention Harness CI modules , You can take help from getting started with Harness blog

Reply:
added the blog reference. thanks